source/common/runtime/runtime_impl.cc

```diff
@@ -23,60 +23,78 @@ namespace Runtime {
 
 const size_t RandomGeneratorImpl::UUID_LENGTH = 36;
 
+void RandomGeneratorImpl::random_bytes(uint8_t* out, size_t out_len) {
```

Comment:
@htuch: we don't, if we don't want to provide generic `randomBytes()` function.

@mattklein123: ah, that wasn't clear to me... anyway, I just removed the whole thing, since you said that it shouldn't be exported via public interface, so there is no point in maintaining the extra code that supported the "generic" part, especially since it required an extra copy.